pandas/core/window.py

```diff
@@ -42,6 +41,7 @@
 from pandas.core.generic import _shared_docs
 from pandas.core.groupby.base import GroupByMixin
 from pandas.core.index import Index, MultiIndex, ensure_index
+from pandas.core.window_.indexers import BaseIndexer
```

Comment:
I see what you are doing here, but I still think ok just to put in pandas.core.window.indexers

Reply:
There's a `pandas/core/window.py` file already, and `from pandas.core.window import ...` is trying to import from the empty `__init__.py` file in this new directory.

I could move `pandas/core/window.py` into `the pandas/core/window/` directory, but I was thinking that would make it harder to merge changes from pandas master.

Reply:
hmm, ok I think we need to move pandas/core/window.py -> pandas/core/window/window.py in pandas main then. we will be needing auxiliary files and it will be simpler. (ok for now here if you want to merge this)